src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -279,7 +282,7 @@ public void bannedUserCanNotEnterTheChat() {
     }
     finally {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move this "finally" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)